### DISCUSSION TITLE
NocoDB on digital ocean [Jul 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2843] [Discord Threads]
> 🙋‍♂️ **Jaxx**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### I assume DO spins down if not active or something.. for this you would have to look up DO docs  
#  
> 🙋‍♂️ **Jaxx**  
#### Thanks, nevermindd I used 'tmux' and used npm start in that  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/997958349662466048)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
